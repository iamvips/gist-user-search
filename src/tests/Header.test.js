import React from "react";
import renderer from "react-test-renderer";
import Header from "../components/Header";

jest.mock("react-redux", () => ({
  ...jest.requireActual("react-redux"),
  useSelector: jest.fn(),
  useDispatch: jest.fn(),
}));

it("should render correctly", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
