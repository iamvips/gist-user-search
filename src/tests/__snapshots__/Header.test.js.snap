// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="sc-hKwDye bUKynq"
  data-testid="header"
>
  <span
    className="mega-octicon octicon-mark-github"
  />
  <div
    className="sc-bdvvtL jcZpGJ"
  >
    <div
      className="sc-gsDKAQ ldqIrX"
    >
      <span
        className="octicon octicon-search"
      />
      <input
        className="sc-dkPtRN lpkptG"
        data-testid="search-input"
        onChange={[Function]}
        placeholder="Search Gists for the username"
      />
    </div>
  </div>
  <span
    className="mega-octicon octicon-home"
    data-testid="home-icon"
  />
</div>
`;
